_posts/2019-01-29-no67-hacking-memories.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Berlin Hack & Tell \#67 - Hacking Memories
+date: 2019-01-29
+time: '18:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/258278925/
+---
+
+* [Minimal Presentation Software for the CLI](https://github.com/jpeinelt/skit) by [Julius Peinelt](https://github.com/jpeinelt)
+* [coline JavaScript command line framework](https://github.com/sebastian-guenther/coline) by [Sebastian](https://sebastianguenther.org)
+* [Shepard tones in Sonic Pi](https://github.com/benjaminwand/Sonic-Pi-Shepard-tones-function)
```

Comment:
add "by [Benjamin Wand](https://github.com/benjaminwand)"

and swap position with previous talk. this was second.